app/controllers/users_controller.rb

```diff
@@ -45,6 +45,13 @@ def update
       end
     end
 
+    unless @user.willing_party
+      (flash[:errors] ||= []).append "You must state which party you are willing to vote for."
```

Comment:
Consider not ending a message with a "." unless it contains multiple sentences.

Reply:
It's pretty tricky to implement given the possibility of what can go into `flash[:errors]`.  Not worth the effort here.

Reply:
Each one is in a separate alert, so they stand alone.

Reply:
Ah you mean a separate red box?

Reply:
Yes